db/migrate_cards/data/progress_bar.json

```diff
@@ -0,0 +1,42 @@
+{
+  "url": null,
+  "timestamp": "2015-05-28 21:03:51 +0100",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[progress bar]]\n[[Project]]\n[[progress bar+*right+*structure]]\n[[project task progress]]\n[[campaign+*type+*structure]]\n[[project with progress bar]]",
+    "value": [
+      {
+        "name": "progress bar",
+        "type": "Cardtype",
+        "codename": "progress_bar"
+      },
+      {
+        "name": "Project",
+        "type": "Cardtype",
+        "codename": "project"
+      },
+      {
+        "name": "progress bar+*right+*structure",
+        "type": "HTML",
+        "content": "<div class=\"progress\">\r\n  <div class=\"progress-bar\" role=\"progressbar\" aria-valuenow=\"{{_l|raw}}\" aria-valuemin=\"0\" aria-valuemax=\"100\" style=\"width: {{_l|raw}}%;\">\r\n    {{_l|raw}}%\r\n  </div>\r\n</div>"
```

Comment:
You're probably right that this is minimal, but it's not super flexible.  If, for example, I wanted to create:
- MyProject+progress on metrics
- MyProject+progress on tasks

I couldn't use that naming pattern.  And in general, I'm stuck with using your name. 

If we go with the type, we can probably stick this in a mod and make it usable in many other contexts.

Also, you probably need to change it so that the editor is like a Phrase (not Basic) editor so that the percentage value doesn't get wrapped in paragraph tags.